pkg/templates/nstemplatetiers/nstemplatetier_generator.go

```diff
@@ -297,11 +298,12 @@ func (t *tierGenerator) createTierTemplates() error {
 	// create the templates
 	for _, tierTmpls := range t.templatesByTier {
 		for _, tierTmpl := range tierTmpls.tierTemplates {
+			log.Info("creating TierTemplate", "tiertemplate", spew.Sdump(tierTmpl))
 			// using the "standard" client since we don't need to support updates on such resources, they should be immutable
 			if err := t.client.Create(context.TODO(), tierTmpl); err != nil && !apierrors.IsAlreadyExists(err) {
 				return errors.Wrapf(err, "unable to create the '%s' TierTemplate in namespace '%s'", tierTmpl.Name, tierTmpl.Namespace)
 			}
-			log.Info("TierTemplate resource created", "namespace", tierTmpl.Namespace, "name", tierTmpl.Name)
+			log.Info("TierTemplate resource created", "namespace", tierTmpl.Namespace, "name", tierTmpl.Name, "tier_name", tierTmpl.Spec.TierName, "revision", tierTmpl.Spec.Revision, "template_name", tierTmpl.Spec.Template.GetName())
```

Comment:
Is there any value in adding the revision to log? I don't see any dependency changes introduced in this PR regarding the revision, maybe I missed it. I don't have an objection to having the revision added to logs, but would like to understand how looking at revision can be helpful to anybody looking at the logs.

Reply:
yeah, that's something I added while working on this PR, but it's not very valuable. Reverted in 7fd75c7